boxsdk/object/item.py

```diff
@@ -322,3 +323,71 @@ def delete(self, params=None, etag=None):
         """
         headers = {'If-Match': etag} if etag is not None else None
         return super(Item, self).delete(params, headers)
+
+
+    def get_from_trash(self, fields=None):
+        """
+        Get item from trash.
```

Comment:
This wording feels confusing. This call gets information. See the inherited get() call which says:
```
        Get information about the object, specified by fields. If fields is None, return the default fields.
```